src/utils/crypto/validation.ts

```diff
@@ -11,12 +13,27 @@ export const isPolicyID = (value: any): boolean =>
 export const isTxHash = (value: any) =>
   !!(typeof value === 'string' && value.length === 64 && isHex(value));
 
-export const isAddress = (value: any) =>
-  !!(
-    typeof value === 'string' &&
-    (value.length === 103 || value.length === 58) &&
-    value.match(/addr1[a-z0-9]+/)
-  );
+export const isEnterpriseAddress = (address: string) => {
+  try {
+    const decodedAddress = CardanoWasmBrowser.ByronAddress.from_bytes(
+      base58.decode(address),
```

Comment:
It's not ByronAddress, it's `EnterpriseAddress`, and you decode it using `EnterpriseAddress.from_bytes(fromHex(hexString))` if you're expecing a hexadecimal string or `EnterpriseAddress.from_bech32(string)` if you're expecting the human readable `addr1...` type of address

Reply:
@MartinSchere actually tell my if I'm wrong but we onyl have two addresses types, Byron and Shelly, Enterprise addresses seem to be a subset inside Shelly Addresses, made the changes in the function names, and also in the validation, feel free to check the updated changes